defi/src/rwa/rwa_start.sh

```
@@ -0,0 +1,81 @@
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+ROOT_DIR=$SCRIPT_DIR/../..
+
+cd $ROOT_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)